website/content/docs/integrations/consul-integration.mdx

```diff
@@ -93,16 +262,45 @@ remaining fields are required to match those shown here.
 }
 ```
 
-</Tab>
-<Tab heading="Consul Enterprise">
+</CodeBlockConfig>
+
+The `JWKSURL` address must be reachable by all Consul servers and should
+resolve to multiple Nomad agents to avoid a single point of failure. Both Nomad
+servers and clients are able to handle this request.
+
+[![Consul Auth Method][img_consul_auth_method]][img_consul_auth_method]
```

Comment:
This doc has a "Server" block that includes both Nomad and Consul. Maybe we should clarify this to be "Nomad Server"?

Reply:
Good point. Added some extra label to each box to indicate what they mean. Not sure if this is the best approach so I will gather some feedback 😅 
![image](https://github.com/hashicorp/nomad/assets/775380/e8247655-fe52-4084-aa99-c11b1775ad1b)

Reply:
I got professional help and it works much better now 😄 
![image](https://github.com/hashicorp/nomad/assets/775380/3a675d83-87d4-4421-ab5a-65ff15d94cf8)